### PR TITLE
Text Updates To Shotguns

### PR DESCRIPTION
**Pull request by DoctorBoomstick**
#### Summary
None

#### Purpose of change
I felt that nearly all of the shotgun non-branded variants, as far as names and descriptions went, were a little lack lustre. When it comes to rifles and pistols, we still give them names and descriptions which give an overview of what the weapon is, such as with the M4A1 being called the M4 carbine, the M27 IAR called the M27 assault rifle, etc. When it came to the shotguns on the other hand, they nearly all had names which referenced their magazine capacities, being labelled as 5-round, 6-round, 7-round, 8-round, shotguns respectively. I felt that this was a little dull in the grand scheme of things and that it clashed with the other established gun names, as in my opinion, if we give semi specific names to the pistols and rifles, why shouldn’t we keep the same pattern with the shotguns.

#### Describe the solution
I have renamed nearly all of the non-branded shotgun variants to names which are still generic, but a bit more descriptive, such as renaming the 6-round shotgun to the “Mossberg assault shotgun”, the KSG to the “12 gauge Kel-Tec bullpup shotgun,” etc. I have also given the same treatment to the descriptions of these weapons. Lastly and unrelated to names and descriptions, the m1014 was referenced in its description as having a collapsing stock, but there was no JSON line adding such an attachment. Given that we now have actual folding stocks, I went ahead and slapped a collapsing wire stock on it.

#### Describe alternatives you've considered
Just leaving them the way that they were.

#### Testing
Their just simple text edits.

#### Additional context
nothing what so ever.

### CODE REVIEW

File: data/json/items/gun/shot.json [hunk 1]
```diff
@@ -102,8 +102,8 @@
     "copy-from": "shotgun_pump",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "14-round tube shotgun" },
-    "description": "This shotgun uses a system of 2 tube magazines to carry 14 rounds of 12 gauge shot.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.",
+    "name": { "str": "12 gauge Kel-Tec bullpup shotgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/shot.json:105]: I'd drop the brand name, it doesn't add anything.
```suggestion
    "name": { "str": "12 gauge bullpup shotgun" },
```

File: data/json/items/gun/shot.json [hunk 2]
```diff
@@ -147,8 +147,8 @@
     "copy-from": "shotgun_pump",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "24-round tube shotgun" },
-    "description": "This shotgun uses a system of 2 tube magazines to carry 24 rounds of 12 gauge shot.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.",
+    "name": { "str": "high-capacity Kel-Tec pump-action shotgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/shot.json:150]: ditto on brand names.
```suggestion
    "name": { "str": "high-capacity pump-action shotgun" },
```

File: data/json/items/gun/shot.json [hunk 3]
```diff
@@ -193,8 +193,8 @@
     "copy-from": "shotgun_pump_3gun",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "6-round shotgun" },
-    "description": "This is a shotgun with capacity for 6 rounds of 12 gauge shot.",
+    "name": { "str": "Mossberg 12 gauge shotgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/shot.json:196]: Again, drop the brand name, but then I feel like this is too generic of a name - hence the original scheme.
```suggestion
    "name": { "str": "12 gauge shotgun" },
```

File: data/json/items/gun/shot.json [hunk 4]
```diff
@@ -262,8 +262,8 @@
     "id": "mossberg_930",
     "copy-from": "shotgun_base",
     "type": "GUN",
-    "name": { "str": "8-round shotgun" },
-    "description": "This is a shotgun with capacity for 8 rounds of 12 gauge shot",
+    "name": { "str": "gas operated semi-automatic shotgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/shot.json:265]: This name feels like too much detail.

File: data/json/items/gun/shot.json [hunk 5]
```diff
@@ -460,8 +461,8 @@
     "id": "remington_870_express",
     "copy-from": "remington_870",
     "type": "GUN",
-    "name": { "str": "7-round shotgun" },
-    "description": "This is a shotgun with capacity for 7 rounds of 12 gauge shot.",
+    "name": { "str": "self-defence pump-action shotgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/shot.json:464]: Again, this name feels like too much detail. I don't want to cut down on what it's saying, but it's just quite long.

File: data/json/items/gun/shot.json [hunk 6]
```diff
@@ -522,8 +523,8 @@
     "copy-from": "shotgun_base",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "Saiga shotgun" },
-    "description": "This shotgun is chambered in 12 gauge, and accepting Saiga magazines.",
+    "name": { "str": "Saiga tactical shotgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/shot.json:526]: This has a brand name because it has corresponding magazines, if you were wondering.

**Review by anothersimulacrum — COMMENTED** [review]
Good work, this is much improved!

**Comment by DoctorBoomstick** [comment]
Thanks for the suggestions, makes sense to not include the brand names now that I think about it. to be honest, working under that logic, I think I’ll also rename the Saiga and its magazines cause, if we’re removing brand names from most of the guns, may as well remove them from all. With regards to the Mossberg and gas-operated stuff, I’ll see what generic old name I can hash out, though the “gas-operated semi-automatic shotgun,” might be a little hard to replace name wise as the non-branded variant includes both the Mossberg 930 and the m1014, which are both very different classes of shotgun.

**Comment by anothersimulacrum** [comment]
Brand names are a problem because they don't provide information. Fictional ones don't solve that. If you want brand names, there is an option for that in interface.

Commentary on merged PRS is quickly lost, further discussion belongs in issues.

**Comment by DoctorBoomstick** [comment]
I'm afraid that I don't quite get what you mean by, "These names make inventory and interface absolutely unreadable.". For reference and complete transparency, I'm fully blind, so I can't really judge the esthetical appearance of text. So, with that in mind, if you could be more specific and explain what's the issue, that'd be appreciated. Is it a case of text being cut off-screen  due to its length, general readability, something else?

**Comment by anothersimulacrum** [comment]
Also, sorry for not getting to this before the merge, @DoctorBoomstick , but I left saiga/think it is fine because it's just an identifier for the magazines that go with the gun. I think what you have is fine too, though.

**Comment by DoctorBoomstick** [comment]
@anothersimulacrum, no problem, I'd got what you meant and figured I'd switch it up any how. to be honest I'm curious to see what the other guy meant when he said that the UI's unreadable, cause I felt like somebody would have hit me up if something like that was an issue. If it's a question of length of the names, that's basically a non-issue, cause I'll just push another pr across with shorter names, but without knowing what exactly he means, I'm hesitant over taking any action.

**Comment by Voker57** [comment]
The problem with long item names like this is you see your inventory or HUD and it's long strings like "An imported russian shotgun" which takes a while to process compared to "Saiga-12" or other short fictional name for which you could once read the description, which still has all the useful information already, and associate a short name with the gun. 

On a related note, stating that a shotgun is imported from a specific country is far from immediately useful information about the gun and should belong in a description.

**Comment by anothersimulacrum** [comment]
Sorry for continuing to talk here, but we seriously need to move this to an issue, this is not an appropriate place for discussion.

**Pull request merged**